scripts/qa_crowbarsetup.sh

```diff
@@ -2341,6 +2341,16 @@ function addfloatingip()
     nova add-floating-ip "$instanceid" "$floatingip"
 }
 
+# by setting --dns-nameserver for subnet, docker instance gets this as DNS info
+# (otherwise it would use /etc/resolv.conf from its host)
+function adapt_dns_for_docker()
+{
+    local partialns=`neutron subnet-show fixed | grep network_id | sed 's/\s//g' | cut -d "|" -f 3`
```

Comment:
@MaximilianMeister this is probably wrong solution, @aplanas is worning on a better one